content/en/agent/logs/advanced_log_collection.md

```diff
@@ -89,7 +89,7 @@ In a Docker environment, use the label `com.datadoghq.ad.logs` on the **containe
 {{% /tab %}}
 {{% tab "Kubernetes" %}}
 
-In a Kubernetes environment, use the pod annotation `ad.datadoghq.com` on your pod to specify the `log_processing_rules`, for example:
+To apply a specific configuration to a given container, Autodiscovery identifies containers by name, NOT image. It tries to match `<CONTAINER_IDENTIFIER>` to `.spec.containers[0].name`, not `.spec.containers[0].image.` To configure your Datadog container logs Autodiscovery on a given `<CONTAINER_IDENTIFIER>` within your pod, add the following annotations to your pod's `log_processing_rules`:
```

Comment:
I don't think "Autodiscovery" is right here, but after a couple minutes of Googling, I haven't been able to figure out what the correct term is. Is it maybe Kubernetes AD? Or maybe just Kubernetes service discovery? The only other reference to Autodiscovery, capitalized, in a Kubernetes context, is Datadog.

Reply:
Hi @urseberry 
We do just refer to it as "Autodiscovery" on the AD integration docs:

https://docs.datadoghq.com/containers/kubernetes/integrations/?tab=kubernetesadv2

But let me know if you think it should be changed to "Kubernetes Autodiscovery" or a link to the Integrations AD?